modules/users/client/controllers/settings/manage-social-accounts.client.controller.js

```diff
@@ -7,7 +7,9 @@ angular.module('users').controller('SocialAccountsController', ['$scope', '$http
     // Check if there are additional accounts
     $scope.hasConnectedAdditionalSocialAccounts = function (provider) {
       for (var i in $scope.user.additionalProvidersData) {
-        return true;
```

Comment:
If it's doing what it looks like doing - checking whether an object has at least one key - I have no idea how to write it in a cleaner way. It's a bigger code design flaw.